src/main/scala/no/ndla/audioapi/service/ConverterService.scala

```diff
@@ -13,32 +13,47 @@ import com.typesafe.scalalogging.LazyLogging
 import com.netaporter.uri.dsl._
 import no.ndla.audioapi.AudioApiProperties._
 import no.ndla.audioapi.auth.User
-import no.ndla.audioapi.model.api.NewAudioMetaInformation
+import no.ndla.audioapi.model.api.NotFoundException
 import no.ndla.audioapi.model.domain.Audio
+import no.ndla.audioapi.model.Language.{AllLanguages, NoLanguage, UnknownLanguage}
 import no.ndla.audioapi.model.{api, domain}
 import no.ndla.mapping.License.getLicense
 
-import scalaj.http.Base64
+import scala.util.{Failure, Success, Try}
+
 
 trait ConverterService {
   this: User with Clock =>
   val converterService: ConverterService
 
   class ConverterService extends LazyLogging {
-    def toApiAudioMetaInformation(audioMetaInformation: domain.AudioMetaInformation): api.AudioMetaInformation = {
-      api.AudioMetaInformation(audioMetaInformation.id.get,
-        audioMetaInformation.titles.map(toApiTitle),
-        audioMetaInformation.filePaths.map(toApiAudio),
-        toApiCopyright(audioMetaInformation.copyright),
-        audioMetaInformation.tags.map(toApiTags))
+    def toApiAudioMetaInformation(audioMetaInformation: domain.AudioMetaInformation, language: String): Try[api.AudioMetaInformation] = {
+      val supportedLanguages = audioMetaInformation.titles
+        .map(_.language.getOrElse(UnknownLanguage))
+        .map(lang => if (lang == NoLanguage) UnknownLanguage else lang)
+        .distinct
+
+      if (supportedLanguages.contains(language) || language == AllLanguages) {
```

Comment:
Jeg tror det er bedre om vi ikke forholder oss til `AllLanguages` med mindre det er snakk om søk uten at språk er definert. Når man henter ut en enkeltressurs er det misledende at det står `"language": "all"` om man ber om en ressurs uten å definere språk. 
Sett følgende eksempel:
```
GET /audio-api/v1/audio/1
{
    "id": 1,
    "language": "all",
    "supportedLanguages": [
        "nn",
        "unknown"
    ],
    ...
}
```

Det at `language` == "all", mens `supportedLanguages` ikke inneholder "all" er litt rart.
Det hadde vært bedre om det i dette tilfellet hadde stått enten en tom streng, eller "unknown" i `language`.
I tillegg må det også være mulig å spørre etter en ressurs på hvilket som helst språk som er definert i `supportedLanguages` (akkurat nå fungerer det ikke å be om `...?language=unknown`).